src/components/bookmark-button/toast/make-bookmark-toast.tsx

```diff
@@ -14,7 +15,7 @@ type ToastActionConfig = Record<
 	{
 		title: string
 		descriptionSuffix: string
-		icon: JSX.IntrinsicElements['svg']
```

Comment:
This is an interface that represents the only props of the `svg` element, not a component accepts the same props as `svg`. I've updated all instances to be `ReactElement<React.JSX.IntrinsicElements['svg']>`